pkgs/os-specific/linux/upower/default.nix

```diff
@@ -80,6 +81,7 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = [
     libgudev
     libusb1
+    polkit
```

Comment:
Looking at the diff, https://gitlab.freedesktop.org/upower/upower/-/commit/0c9ba8048fd38567aa1292f3760fbff48904c3ca sets `POLKIT_ACTIONDIR` define but it does not look like it is currently used? If not, that should not be an issue.

Reply:
Indeed, it doesn't look like it's used.